docs/docs/guides/manage/user/scim2.md

```diff
@@ -0,0 +1,290 @@
+---
+title: SCIM v2.0
+---
+
+:::info
+The SCIM v2 interface of Zitadel is currently in a [preview stage](/support/software-release-cycles-support#preview).
+It is not yet feature-complete, may contain bugs, and is not generally available.
+
+Do not use it for production yet.
```

Comment:
```suggestion
Do not use it for production yet.

As long as the feature is in a preview state, it will be available for free, it will be put behind a commercial license once it is fully available.
```